src/clusterfuzz/_internal/bot/fuzzers/init.py

```diff
@@ -29,7 +30,7 @@ def run():
   """Initialise builtin fuzzing engines."""
   engine.register('afl', afl_engine.AFLEngine)
   engine.register('blackbox', blackbox_engine.BlackboxEngine)
-  engine.register('gft', gft_engine.GFTEngine)
+  engine.register('googlefuzztest', gft_engine.GoogleFuzzTestEngine)
```

Comment:
I'm of the opinion that CF should consider every engine's name to be all lowercase.
So classes for googlefuzztest should be named like "GooglefuzztestEngine" instead of GoogleFuzzTestEngine.
I think it's slightly inconsistent to call it googlefuzztest in some places but then call it GoogleFuzzTest in others.

Reply:
I think this inconsistency is OK. The string name is just an arbitrary display name for the user that doesn't have to correlate to the implementation's name. There's no reason why the Python class casing should be a strict casing according to this . 

Re auto searching directory -- we could do this, but adding a new engine is a once in a year (or more)  event so may not be worth the complexity at the moment.